### DISCUSSION TITLE
How to use Component Story Format 3 with Stencil or Lit?

### DISCUSSION THREAD

**Discussion by JangoCG** [storybookjs/storybook#23563] [Help]
I cannot find any docs or working examples where stencil or lit is used with the new component story format 3? 
Does anyone here have an working example?

**Comment by jonniebigodes** [comment]
Hey @JangoCG, thanks for reaching out to us with your question. Appreciate it 🙏 ! We currently don't have any available Stencil documentation, but for Lit, have you tried selecting the Web Components framework from the framework picker at the top of the page on our documentation?

**Reply by break-stuff** [reply]
This option will also work for Stencil and I recommend it.

**Comment by break-stuff** [comment]
Here's a demo I did for Stencil. I will see if I can whip one up for Lit.

https://github.com/break-stuff/stencil-storybook-demo

**Reply by gethari** [reply]
By any chance did you make one for lit ? Can you please share if so ?

**Reply by break-stuff** [reply]
Yeah, here is an example I did for a workshop. You can look at the "complete" branch with a sample component.

https://github.com/break-stuff/web-components-workshop
